### PR TITLE
fix rt.sh for 35 day tests

### PR DESCRIPTION
**Pull request by MinsukJi-NOAA**
This PR address issue #156 

When running 35 day tests, the recommended command is `rt.sh -l rt_35d_*.conf -cek`:
  - `RTPWD` does not have to be changed to 20200530
  - Results will be in `RUNDIR`

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
This appears to be working correctly for my CICE6 branch. The 20120101 cold start file was copied into the REGRESSION_TEST directory called MEDIATOR_bm_cmeps_2012010100; it was copied from there into the 35d run directory.

**Comment by MinsukJi-NOAA** [comment]
> This appears to be working correctly for my CICE6 branch. The 20120101 cold start file was copied into the REGRESSION_TEST directory called MEDIATOR_bm_cmeps_2012010100; it was copied from there into the 35d run directory.

That is correct.  Although `-c` flag means create a new baseline (i.e. REGRESSION_TEST directory), this is just a dummy (or intermediate) location to transfer new mediator files from the cold runs to warm runs. For test results, you should go back to the run directory (and not REGRESSION_TEST directory); this is why you also need `-k` flag to not delete the RUNDIR.

File: tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d [hunk 1]
```diff
@@ -12,8 +12,8 @@ export SMONTH
 export SDAY="01"
 export SHOUR="00"
 
-export DAYS="35"
-export FHMAX="840"
+export DAYS="2"
```
Comment (BrianCurtis-NOAA) [inline tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d]: Should this be put back to 35 and 840?

Comment (MinsukJi-NOAA) [reply tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d]: @DeniseWorthen Should we change this back to 35 days?

Comment (DeniseWorthen) [reply tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d]: Yes before we commit, we need to change this back. It was only used to be able to create a short test vs running the whole 35d.

I want to do one more test w/ my cice6 branch, which is to run more than 1 date.  I expect it to work. 

I haven't done the ww3 35d test though. Did you test that Minsuk?

Comment (MinsukJi-NOAA) [reply tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d]: I have run 35 day (well, 48 hours) tests for all dates with and without ww3.

Comment (DeniseWorthen) [reply tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d]: OK, great. It worked for more than one day in my cice6 branch too so I think it is good.

**Review by DeniseWorthen — APPROVED** [review]

**Comment by BrianCurtis-NOAA** [comment]
Testing this feature:
closes #156

**Review by junwang-noaa — APPROVED** [review]

**Pull request merged**